requirements_dev.txt

```diff
@@ -1,2 +1 @@
-homeassistant==2023.12.1
-ffmpeg
```

Comment:
This `ffmpeg` ***Python module*** ([PyPi link](https://pypi.org/project/ffmpeg/)) is of course not a dependency of Versatile Thermostat. I gather that it is provides a Python wrapper around the `ffmpeg` binary, but the binary must still be separately installed in the devcontainer. If HASS itself needs such a wrapper, then it is a dependency of HASS and it does not need to be included here.